suite-native/message-system/src/components/MessageSystemRenderer.tsx

```diff
@@ -1,39 +1,52 @@
+import { ReactNode } from 'react';
 import { useSafeAreaInsets } from 'react-native-safe-area-context';
 import { useSelector } from 'react-redux';
 
 import { A } from '@mobily/ts-belt';
 
 import { Box, VStack } from '@suite-native/atoms';
 import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
-import { selectActiveBannerMessages } from '@suite-common/message-system';
+import {
+    selectActiveBannerMessages,
+    selectActiveFeatureMessages,
+} from '@suite-common/message-system';
 
 import { MessageBanner } from './MessageBanner';
+import { FeatureMessageScreen } from './FeatureMessageScreen';
 
 const messageSystemRendererStyle = prepareNativeStyle<{ topSafeAreaInset: number }>(
     (_, { topSafeAreaInset }) => ({
         marginTop: topSafeAreaInset,
     }),
 );
 
-export const MessageSystemRenderer = () => {
+type MessageSystemRendererProps = {
+    children: ReactNode;
+};
+
+export const MessageSystemRenderer = ({ children }: MessageSystemRendererProps) => {
     const { applyStyle } = useNativeStyles();
     const { top: topSafeAreaInset } = useSafeAreaInsets();
 
     const activeBannerMessages = useSelector(selectActiveBannerMessages);
-
-    if (A.isEmpty(activeBannerMessages)) return null;
+    const activeFeatureMessages = useSelector(selectActiveFeatureMessages);
+    const firstFeatureMessage = A.head(activeFeatureMessages);
 
     return (
-        <Box
-            style={applyStyle(messageSystemRendererStyle, {
-                topSafeAreaInset,
-            })}
-        >
-            <VStack spacing={4}>
-                {activeBannerMessages.map(message => (
-                    <MessageBanner key={message.id} message={message} />
-                ))}
-            </VStack>
-        </Box>
+        <>
+            {children}
```

Comment:
{children} should be rendered after the `activeBannerMessages`. Now it is other way around.

Reply:
this is how the code in develop rendered (not visible):
<img width="542" alt="Screenshot 2023-11-10 at 21 51 51" src="https://github.com/trezor/trezor-suite/assets/2011829/fd1ff1f5-542a-4b82-90cf-b19670c3d4fb">

so I changed it and this is how it renders now:
<img width="539" alt="Screenshot 2023-11-10 at 21 54 10" src="https://github.com/trezor/trezor-suite/assets/2011829/12ac82ae-2882-4077-87bb-fcb66d7ec1c8">

Reply:
Martin is correct here, what comes last has higher zIndex eg it will be on top.

Reply:
Ok, my bad then. If the banner messages are rendered above the rest of the screen UI, I'm happy :).